include/functions.h

```diff
@@ -1040,8 +1040,8 @@ s32 Health_ChangeBy(GlobalContext* globalCtx, s16 healthChange);
 void Rupees_ChangeBy(s16 rupeeChange);
 void Inventory_ChangeAmmo(s16 item, s16 ammoChange);
 void Magic_Fill(GlobalContext* globalCtx);
-void func_800876C8(GlobalContext* globalCtx);
-s32 func_80087708(GlobalContext* globalCtx, s16 arg1, s16 arg2);
+void Magic_Reset(GlobalContext* globalCtx);
+s32 Magic_RequestChange(GlobalContext* globalCtx, s16 magicChange, s16 changeType);
```

Comment:
arg names need to be updated
I also think its appropriate to change `Health_ChangeBy` to use `amount` for its arg with these set of changes